.github/workflows/main.yml

```diff
@@ -526,7 +526,7 @@ jobs:
           python3 -m pip install wheel
           python3 -m pip install future
       - name: configure
-        run: cmake $superbuild $cmake_prefix_path -DENABLE_STRICT_TRY_COMPILE=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=$(pwd)/tools/ios.toolchain.cmake -DPLATFORM=${{ matrix.platform }} -DDEPLOYMENT_TARGET=13.0 -DBUILD_MAVSDK_SERVER=ON -DBUILD_SHARED_LIBS=OFF -DWERROR=OFF -Bbuild/${{ matrix.name }} -H.
+        run: cmake $superbuild $cmake_prefix_path -DENABLE_STRICT_TRY_COMPILE=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=$(pwd)/tools/ios.toolchain.cmake -DPLATFORM=${{ matrix.platform }} -DDEPLOYMENT_TARGET=13.0 -DBUILD_MAVSDK_SERVER=ON -DBUILD_SHARED_LIBS=OFF -DWERROR=OFF -Bbuild/${{ matrix.name }} -DPython_FIND_STRATEGY=LOCATION -DPython3_EXECUTABLE=$(command -v python3) -H.
```

Comment:
I hate this line :laughing:. But I guess we don't really have a choice here if we want CMake to detect python.

Reply:
But this line is mostly just for CI, so it can be ugly. If only it worked though...

Reply:
Seems like it does!

Reply:
Oh I did it! Holy moly.

Reply:
Let's see if we can without that and just the "don't use framework" option.